cmd/kubectl-testkube/commands/common/repository.go

```diff
@@ -8,6 +8,9 @@ import (
 
 // NewRepositoryFromFlags creates repository from command flags
 func NewRepositoryFromFlags(cmd *cobra.Command) (repository *testkube.Repository, err error) {
+	if cmd.Flag("git-uri") == nil {
```

Comment:
why did we add it? we callit from somewhere without this flag?

Reply:
yes, we call it from generate cmd, and it doesn't have this param, maybe we can put a check where this func is called.